x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_content/log_rate_analysis_attachments_menu.tsx

```diff
@@ -71,6 +95,19 @@ export const LogRateAnalysisAttachmentsMenu = () => {
     [dataView.id, embeddable, applyTimeRange, timeRange]
   );
 
+  const caseAttachmentTooltipContent = useMemo(() => {
+    if (!showLogRateAnalysisResults) {
+      return i18n.translate('xpack.aiops.logRateAnalysis.attachToCaseTooltipNoAnalysis', {
+        defaultMessage: 'Run the analysis first to add results to a case',
```

Comment:
```suggestion
        defaultMessage: 'Run the analysis first to add results to a case.',
```

Reply:
Done in: [#33d5e29](https://github.com/elastic/kibana/pull/201549/commits/33d5e29150c5734b147d73622a8010190c41aa3e)